doc/releases/changelog-dev.md

```diff
@@ -68,3 +71,4 @@ Lillian Frederiksen
 Andrija Paurevic,
 Korbinian Kottmann,
 Christina Lee
+Po-Ying Chiu
```

Comment:
The order is alphabetical according to last name, so it should be the second name in the list.